www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: layouts/post.njk
+---
+
+After 40+ beta versions & release candidates we are so excited to introduce **Snowpack 2.0**, featuring:
+
+- A bundle-free dev environment that starts up in 50ms or less.
+- Application build tooling that stays fast as your codebase grows.
+- [Bundled production-optimized builds](/#snowpack-build) (powered by Webpack or Parcel)
+- [Hot Module Replacement (HMR)](/#hot-module-replacement) for React, Preact, Vue, Svelte & more.
+- Out-of-the-box support for TypeScript, JSX, CSS Modules & more.
+- [Custom build scripts](/#build-scripts) & [third-party plugins](/#build-plugins) to connect your favorite tools.
+
+For most Snowpack v1.x applications, all you need to do is run the following command and Snowpack 2.0 will walk you through updating any outdated configuration:
+
+```
+npm i snowpack@latest --save-dev
+```
+
+
+<br/>
+
+## Snowpack 2.0: Rethinking Best Practices
+
+!["Rethink established best practices" tweet.](/img/react-snarky-tweet-2.png)
+
+"Rethink established best practices." That was the famous antagonistic tweet directed at the React team back when React & JSX were first announced in 2013.
+
+It had been meant as a snarky burn against the project, but the React team turned it into a rallying cry. Pete Hunt even named [an entire JSConf talk](https://www.youtube.com/watch?v=DgVS-zXgMTk) after it, explaining why "rethinking best practices" was exactly what the web needed at that moment. 7 years later, and Pete was clearly proven right.
```

Comment:
"it had been meant" is a little clunky, lots of extra words in this first sentence. "It was meant as a burn at React, but the team turned it into a rallying cry"

"7 years later, and pete was clearly proven right" -> "7 years later, Pete was proven correct"

Reply:
maybe "The Tweet was a criticism of React, but ..."

Reply:
I also think you could remove "exactly", and "exactly" is used a few other places in the post as well